help_to_heat/utils.py

```diff
@@ -240,11 +240,12 @@ def get_current_and_next_month_names(month_names):
     return current_month, next_month
 
 
-# Q1 january -> april
-# Q2 april -> july
-# Q3 july -> october
-# Q4 october -> january
+# Q1 january -> may
+# Q2 april -> august
+# Q3 july -> november
+# Q4 october -> february
 # we update the month names manually on each scottish data dump
 # this is to tell the user the month of the last dump, and expected month of next dump
+# the quarters overlap by one month, as the dump takes place roughly at the end of a month
 def get_current_and_next_quarter_month_names(month_names):
-    return month_names[6], month_names[9]
+    return month_names[9], month_names[1]
```

Comment:
I think this language is confusing now - Q1 is not January-May for example. Worth reviewing: what are we actually trying to say with these?

Are we definitely right to say October?

Reply:
Hmm, you're right. I'll have a rethink of the wording.

I think what I'm trying to get across with this comment is which pairs of months to use in which quarter, ultimately. But yeah there's a disconnect between the quarter we're *in*, and the quarter of the data dump you're performing.

Reply:
I've attempted to be clearer about the purpose of the comment, and how it should be used. I've also renamed the function to be a bit closer to what it actually is doing.